package.json

```diff
@@ -29,15 +41,9 @@
     "getstream.io"
   ],
   "files": [
-    "/dist",
-    "/src",
-    "readme.md",
-    "license"
+    "/dist"
```

Comment:
From [npm documentation](https://docs.npmjs.com/cli/v11/configuring-npm/package-json#files): 

Certain files are always included, regardless of settings:

- package.json
- README
- LICENSE / LICENCE
- The file in the "main" field
- The file(s) in the "bin" field

Not sure why we were releasing sourcefiles as well though.

Reply:
Sources are good convenience... I personally like it when they are around, as I can easily poke around with my IDE instead of using GitHub or clone the repo and make sure I'm on the correct tag.

Reply:
I rarely check the `dist`'s for sources but I'll return them if you think it's convenience. 👍